alpha/scripts/utils/getWorkersStatus.php

```diff
@@ -0,0 +1,83 @@
+<?php
+ob_start();
+// Report simple running errors only(no notice!!)
+error_reporting(E_ERROR | E_WARNING | E_PARSE);
+require_once(__DIR__ . '/../bootstrap.php');
+
+/**
+ * $argv[1] - path to files configuration folder
+ * example: /opt/kaltura/app/configurations/batch
+ * $argv[2] - name of candidates example workers.ini 
+ */
+$files = glob($argv[1]."/*$argv[2]");
+$answers = array();
+foreach($files as $file)
+{
+	try
+	{
+		$config = new Zend_Config_Ini($file);
+	}
+	catch (Zend_Config_Exception $e)
+	{
+		die($e);
+	}
+	$configArray = $config->toArray();
+	$periodicWorkers = getPeriodicWorkers($configArray);
+	foreach ($periodicWorkers as $worker)
+	{
+		$criteria = new Criteria(SchedulerWorkerPeer::DATABASE_NAME);
+		$criteria->add(SchedulerWorkerPeer::CONFIGURED_ID, $worker['id']);
+		try
+		{
+			$schedulerWorkers = SchedulerWorkerPeer::doSelect($criteria);
+		}
+		catch (PropelException $e)
+		{
+			die($e);
+		}
+		/** @var SchedulerWorker  $schedulerWorker */
+		foreach ($schedulerWorkers as $schedulerWorker)
+		{
+			$lastExecutionTime = $schedulerWorker->getStatuses()[SchedulerStatus::RUNNING_BATCHES_LAST_EXECUTION_TIME];
+			$sleepBetweenStopStart = $worker['sleepBetweenStopStart'];
+			if ($lastExecutionTime)
+			{
+				/** $lastExecutionTime + max($sleepBetweenStopStart , 300 )  we are letting a margin of twice the execution time meaning
+				 *  if job a should run every hour
+				 *  we will wait for two hours before printing it as error
+				 * minimum is 5 minutes so if job should run every 60 sec will will only inform the error after 5 minutes
+				 */
+				if ($lastExecutionTime + max($sleepBetweenStopStart , 300 ) < time() - $worker['sleepBetweenStopStart'])
```

Comment:
Move 300 to be external param - Margin time
Use sleepBetweenStopStart  instead of  $worker['sleepBetweenStopStart']